<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frameBlue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>410</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>GestureBoard</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescription">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Leírás</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCvImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameButtons">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>520</width>
     <height>410</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lblCamera">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartCam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamerateszt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Vissza</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderHue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>95</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblHue">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Hue eltolás</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFrameCnt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Képkocka gesztusonként</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDelay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Szünet gesztusok között</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConfidence">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Magabiztosság</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinConfidence">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinFrameCnt">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinDelay">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Mentés</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboCamera">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
